homeassistant/components/deconz/diagnostics.py

```diff
@@ -0,0 +1,39 @@
+"""Diagnostics support for deCONZ."""
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.system_info import async_get_system_info
+
+from .gateway import get_gateway_from_config_entry
+
+
+async def async_get_config_entry_diagnostics(
+    hass: HomeAssistant, config_entry: ConfigEntry
+) -> dict[str, Any]:
+    """Return diagnostics for a config entry."""
+    gateway = get_gateway_from_config_entry(hass, config_entry)
+    diag: dict[str, Any] = {}
+
+    diag["home_assistant"] = await async_get_system_info(hass)
```

Comment:
hass system info is added central with #64670, so it can be removed here.

Reply:
I will put up a PR for that once #64670 has been merged.

Reply:
Fixed https://github.com/home-assistant/core/pull/64704